tests/foreman/destructive/test_katello_certs_check.py

```diff
@@ -106,7 +107,7 @@ def test_positive_install_sat_with_katello_certs(certs_vm_setup):
     """
     cert_data, rhel_vm = certs_vm_setup
     version = rhel_vm.os_version.major
-    rhel_vm.download_repos(repo_name='satellite', version=version)
+    Satellite(rhel_vm.hostname).download_repofile()  # as we need satellite repos
```

Comment:
https://github.com/SatelliteQE/robottelo/pull/9986#discussion_r952451809

Reply:
This should be resolved by #10079.
I would be grateful for a review.